src/LndClient/Data/Newtype.hs

```diff
@@ -251,3 +270,13 @@ defaultSyncGrpcTimeout = GrpcTimeoutSeconds 60
 
 defaultAsyncGrpcTimeout :: GrpcTimeoutSeconds
 defaultAsyncGrpcTimeout = GrpcTimeoutSeconds 3600
+
+toSat :: MSat -> Either LndError Sat
+toSat mSat = do
```

Comment:
Here toSat returns ToGrpcError as it preferably is used to transform MSat to Sat before grpc call
and it needs for composition with toGrpc